validation/src/test/scala/hmda/validation/rules/lar/quality/Q005Spec.scala

```diff
@@ -0,0 +1,43 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q005Spec extends LarEditCheckSpec with BadValueUtils {
+  property("All generated lars must pass") {
+    forAll(larGen) { lar =>
+      lar.mustPass
+    }
+  }
```

Comment:
`loan.amount` is created with `Gen.posNum[Int]`, which in all the tests I've seen, never generates a number higher than 99.  I don't know if that's behavior that we've defined.